### PR TITLE
Feature/ruby 2.6

### PR DESCRIPTION
**Pull request by JohnBDonner**
Jira task: [BP-6472]

Updating the gem to live in ruby 2.6.5

Also updated circleci config to use Fei's gem orb

[BP-6472]: https://blinkist.atlassian.net/browse/BP-6472

### CODE REVIEW

File: blinkist-airbrake-scrubber.gemspec [hunk 1]
```diff
@@ -8,15 +9,15 @@ Gem::Specification.new do |gem|
   gem.version       = Blinkist::AirbrakeScrubber::VERSION
   gem.authors       = ["Paweł Komarnicki", "Dinesh Vasudevan", 'Tomek Przedmojski']
```
Comment (zfhui) [inline blinkist-airbrake-scrubber.gemspec:10]: While you are at it, do you think we should also change Pawel to `Blinkist` and the email to operations@blinkist.com?

File: .circleci/config.yml [hunk 2]
```diff
@@ -1,91 +1,49 @@
-# Ruby CircleCI 2.0 configuration file
-#
-# Check https://circleci.com/docs/2.0/language-ruby/ for more details
-#
-version: 2
+version: 2.1
+orbs:
+  gem: zfhui/ruby-gem@0.1.0
```
Comment (zfhui) [inline .circleci/config.yml:3]: Orb zfhui/ruby-gem@0.2.1 is out, with which you can also release a gem to RubyGems (workin on the Gemfury part :) ). Feel free to give it a go, if you want, but I'm also fine with this cleaning up.

**Comment by JohnBDonner** [comment]
Just updated the `RUBYGEMS_API_KEY` on circleCI to be the current API key from our rubygems account 👍

File: .circleci/config.yml [hunk 3]
```diff
@@ -1,91 +1,43 @@
-# Ruby CircleCI 2.0 configuration file
-#
-# Check https://circleci.com/docs/2.0/language-ruby/ for more details
-#
-version: 2
+version: 2.1
+orbs:
+  gem: zfhui/ruby-gem@0.2.1
```
Comment (dinks) [inline .circleci/config.yml:3]: May be in the future could we do this like `blinkist/ruby-gem`?

File: .circleci/config.yml [hunk 4]
```diff
@@ -1,91 +1,43 @@
...
-
+      - gem/setup-and-test:
+          gemspec: blinkist-airbrake-scrubber.gemspec
+          bundler-version: 2.0.1
+  test-with-ruby-2-3-6:
     working_directory: ~/repo
-
+    executor:
+      name: gem/default
+      tag: 2.3.6
```
Comment (dinks) [inline .circleci/config.yml:18]: This does not change anything. I think we need to test with `2.6.5`

File: blinkist-airbrake-scrubber.gemspec [hunk 5]
```diff
@@ -1,22 +1,23 @@
...
 require_relative "lib/blinkist-airbrake-scrubber/version"
 
 Gem::Specification.new do |gem|
   gem.name          = "blinkist-airbrake-scrubber"
   gem.version       = Blinkist::AirbrakeScrubber::VERSION
-  gem.authors       = ["Paweł Komarnicki", "Dinesh Vasudevan", 'Tomek Przedmojski']
-  gem.email         = ["pawel@blinkist.com", "dinesh@blinkist.com", "tomek@blinkist.com"]
-  gem.description   = %q{Email scrubbing configuration for Airbrake at Blinkist}
-  gem.summary       = %q{With this, Airbrake will not leak emails via exception notifications}
+  gem.authors       = ["Blinkist", "Dinesh Vasudevan", 'Tomek Przedmojski']
```
Comment (dinks) [inline blinkist-airbrake-scrubber.gemspec:10]: Let's keep only `Blinkist`?

File: blinkist-airbrake-scrubber.gemspec [hunk 6]
```diff
@@ -1,22 +1,23 @@
...
+  gem.email         = ["operations@blinkist.com", "dinesh@blinkist.com", "tomek@blinkist.com"]
```
Comment (dinks) [inline blinkist-airbrake-scrubber.gemspec:11]: Again only `operations`

File: .circleci/config.yml (hunk 3 above)
Comment (JohnBDonner) [reply .circleci/config.yml:3]: Agreed, I think that's Fei's intention.

**Pull request merged**